src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,243 @@
+package duke;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.time.DateTimeException;
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.util.ArrayList;
+import java.util.List;
+
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.Todo;
+
+/**
+ * Deals with loading tasks from save file and saving tasks in the file.
+ */
+public class Storage implements Storable {
+
+    /** Tasks saved are separated by SAVE_SEPARATOR */
+    private static final String SAVE_SEPARATOR = " ~ ";
+
+    /** Filename of file to be saved to */
+    private final String saveFileName;
```

Comment:
This variable sounds a little like a verb, maybe you could change it to be a noun?

Reply:
Thanks for the suggestion! However since "separator" is a noun and that it sufficiently describes the purpose of this constant, I do not feel that it is necessary to make changes to this name.